### ISSUE TITLE
what board am I API endpoint

### ISSUE THREAD

**Issue by BLuedtke** [uniba-swt/swtbahn-cli#131]
Add an endpoint that returns the name of the model railway (e.g., swtbahn-full), here called "board" or "platform". Not to be confused with a "bidib board" or a platform in the sense of a station platform.

This name is currently not defined in the config files; one workaround could be to use the name of the folder the config files are located in.

**Comment by BLuedtke** [comment]
Once this is done, open a new issue for adjusting the "game" client accordingly.

**Comment by BLuedtke** [comment]
This also has implications for the experimental-point-control branch/map controller, as that is currently hardcoded to swtbahn-full -> can be made more flexible.

**Comment by BLuedtke** [comment]
The config.bahn file actually has the info we want - it starts with `module <name>`, where `<name>` is (by convention) the name of the board/platform, e.g., `SWTbahnFull` as can be seen [here](https://github.com/uniba-swt/swtbahn-cli/blob/master/configurations/swtbahn-full/config.bahn#L1).

However, this name of the "module" in the BahnDSL config is not included by the BahnDSL compiler/generator in any of the other files, except for in form of a comment in the first line of a file (see [this for example](https://github.com/uniba-swt/swtbahn-cli/blob/master/configurations/swtbahn-full/bidib_track_config.yml#L1)). 

One option would be to parse the comment of one (or more?) of the yaml config files, before starting the parser for the actual yaml contents. But this is pretty fragile and relies on the comment being there; a user (e.g., a new hiwi) would not expect that changing a comment in a yaml file could change how the system behaves. 

I can see two valid options: Either the name of the model railway being "served" has to be provided as a command line parameter when starting the server, or we must change BahnDSL to include the module name, in e.g., the extras_config.yml. However, I'm not sure how much work this would cause in BahnDSL, as I'm not too familiar with that codebase. I'll have a look and then decide.

**Comment by BLuedtke** [comment]
I got BahnDSL dev environment set up, and prototyped a little. Its pretty easy to add the name of the module defined in config.bahn in any of the yaml outputs. For the prototype, I put it into the extras_config.yaml like `modulename: SWTbahnFull` in the first line after the comment. This could be parsed by the parser in the swtbahn cli and made accessible with reasonable effort.

The problem: I'm not a maintainer for BahnDSL (thats Tri and Eugene only) and I have no idea how to do these releases and so on. Might ask Eugene how we want to resolve that.

**Comment by eyip002** [comment]
You can make a pull request over in https://github.com/trinnguyen/bahndsl and let @trinnguyen or I merge it into master. I'm only a contributor so @trinnguyen has to give you permission to do releases.

**Comment by trinnguyen** [comment]
Hey @BLuedtke, I've invited you to collaborate on the BahnDSL repository. Please check your email. It would allow you to push the code and release a new version. If you need help reviewing the code before merging as a double check, send me or @eyip002 a PR.

**Comment by BLuedtke** [comment]
Thanks a lot!

**Comment by eyip002** [comment]
Linked branch has been merged.
